binding/angular/projects/rhino-angular/karma.conf.js

```diff
@@ -25,7 +25,7 @@ module.exports = function (config) {
       suppressAll: true, // removes the duplicated traces
     },
     coverageReporter: {
-      dir: require('path').join(__dirname, '../../coverage/rhino-web-angular'),
+      dir: require('path').join(__dirname, '../../coverage/rhino-angular'),
```

Comment:
both the binding and demo have a bunch of bloat with these `karma` and `jasmine` testing libraries. We're not using them at all and we can add them at a later date if we choose to add tests. Can you remove `karma` and `jasmine` libraries from `package.json` and remove this `karma.conf.js` file?